src/openrct2/platform/platform.h

```diff
@@ -124,6 +124,7 @@ uint8_t platform_get_locale_temperature_format();
 uint8_t platform_get_locale_date_format();
 bool platform_process_is_elevated();
 bool platform_get_steam_path(utf8* outPath, size_t outSize);
+bool is_running_in_wine();
```

Comment:
platform.h is a legacy file. Please use Platform2.h instead. We are in the process of transitioning, sorry about that…

A version of this function will need to be implemented for other platforms, too, even if it just returns false. I think you can put it in Platform.Posix.cpp.